go.mod

```diff
@@ -571,4 +573,5 @@ replace (
 	github.com/facebookincubator/nvdtools => github.com/stackrox/nvdtools v0.0.0-20231111002313-57e262e4797e
 	github.com/heroku/docker-registry-client => github.com/stackrox/docker-registry-client v0.2.0
 	github.com/mholt/archiver/v3 => github.com/anchore/archiver/v3 v3.5.2
+	github.com/quay/claircore => github.com/quay/claircore v1.5.34-0.20241217183852-82f8e60f340e
```

Comment:
I don't think we need this. Just update the current version with `main` then run `go mod tidy`. I'm sure we'll aim to cut a new release later this month